plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/internal/ResultSetMessages_ro_RO.properties

```diff
@@ -0,0 +1,463 @@
+controls_resultset_viewer_action_view_format = Vizualizare/Format
+controls_resultset_viewer_action_view_as = Vizualizare ca
+controls_resultset_viewer_action_view_column_type = Seta\u0163i transformarea "{0}".
+controls_resultset_viewer_action_view_column_types = Coloana se transform\u0103...
+controls_resultset_viewer_action_data_formats = Formate de date...
+controls_resultset_viewer_action_binary_format = Format binar
+controls_resultset_viewer_action_panels = Panouri
+controls_resultset_viewer_action_refresh = Actualizeaz\u0103 setul de rezultate
+controls_resultset_viewer_action_navigate = Naviga\u0163i
+controls_resultset_viewer_action_layout = Aspect
+controls_resultset_viewer_add_new_row_context_name = Copia\u0163i valorile r\u00E2ndurilor
+controls_resultset_viewer_dialog_status_title = Stare
+controls_resultset_viewer_job_update = Actualiza\u0163i datele
+controls_resultset_viewer_monitor_aply_changes = Aplica\u0163i modific\u0103rile setului de rezultate
+controls_resultset_viewer_status_empty = Gol
+controls_resultset_viewer_status_inserted_ = Inserat: {0} / \u015Eters: {1} / Actualizat: {2}
+controls_resultset_viewer_status_no_data = Nu exist\u0103 date
+controls_resultset_viewer_status_row = R\u00E2nd
+controls_resultset_viewer_status_rows_fetched = {0} r\u00E2nd(e) preluat(e) {1}
+controls_resultset_viewer_status_rows_updated = {0} r\u00E2nd(e) actualizat(e) {1}
+controls_resultset_viewer_status_rows_size = {0} r\u00E2nduri (+{1})
+
+controls_resultset_viewer_status_rows_time_long = - {0}, pe {1}
+controls_resultset_viewer_status_rows_time_fetch_long = - {0} ({1} preluare), pe {2}
+controls_resultset_viewer_status_rows_time = - {0}, pe {1} la {2}
+controls_resultset_viewer_status_rows_time_fetch = - {0} ({1} preluare), pe {2} la {3}
+controls_resultset_viewer_value = Valoare
+controls_resultset_viewer_calculate_row_count = Calcula\u0163i num\u0103rul total de r\u00E2nduri
+controls_resultset_viewer_pin_column = Fixa\u0163i coloana "{0}"
+controls_resultset_viewer_pin_columns = Fixa\u0163i coloanele ({0})
+controls_resultset_viewer_unpin_column = Anula\u0163i fixarea coloanei "{0}"
+controls_resultset_viewer_unpin_columns = Anula\u0163i fixarea coloanelor ({0})
+controls_resultset_viewer_hide_column_x = Ascunde coloana "{0}"
+controls_resultset_viewer_hide_columns_x = Ascunde\u0163i coloanele ({0})
+controls_resultset_viewer_hide_columns_error_title = Nu se pot ascunde coloanele
+controls_resultset_viewer_hide_columnss_error_text = Nu se pot ascunde toate coloanele cu rezultate: cel pu\u0163in o coloan\u0103 trebuie s\u0103 fie vizibil\u0103
+controls_resultset_viewer_show_hidden_columns = Afi\u015Feaz\u0103 coloanele ascunse
+
+controls_resultset_ref_menu_no_references=<No References>
+controls_resultset_ref_menu_references=<Table References>
+controls_resultset_ref_menu_no_associations=<No Foreign Keys>
+controls_resultset_ref_menu_associations=<Table Foreign Keys>
```

Comment:
You can localize it too, if you want.

Reply:
At the moment I would prefer to keep them